docs/source/api/python/mantid/api/SpectrumInfo.rst

```diff
@@ -4,6 +4,165 @@
 
 This a python binding to the C++ class Mantid::API::SpectrumInfo.
 
+--------  
+Purpose 
+-------- 
+The purpose of the ``SpectrumInfo`` object is to allow the user to access information about the spectra being used in an experiment. The ``SpectrumInfo`` object can be used to access information such as the number of spectra, the absolute position of a spectrum as well as the distance from the sample to the source. There are many other methods available as well. 
+
+A Spectrum corresponds to (a group of) one or more detectors. However if no instrument/ beamline has been set then it could also technically be zero or more detectors. An example test case details this below.  
+ 
+Many users may need this extra information so that they can have a better understanding of the beamline they are using. This extra information is easy and fast to access. 
+
+``SpectrumInfo`` is one of three objects that the user can gain access to from a workspace. 
+The other two are:
+ * ``DetectorInfo``
+ * ``ComponentInfo``
+
+------
+Usage
+------
+
+**Example 1 - Creating a SpectrumInfo Object:**
+This example shows how to obtain a ``SpectrumInfo`` object from a workspace object.
+The return value is a ``SpectrumInfo`` object.
+
+.. testcode:: CreateSpectrumInfoObject
+	
+	# Create a workspace to use
+	ws = CreateSampleWorkspace()
+
+	# Get the SpectrumInfo object
+	info = ws.spectrumInfo()
+	print(type(info))
+
+Output:
+
+.. testoutput:: CreateSpectrumInfoObject
+
+	<class 'mantid.api._api.SpectrumInfo'>
+
+
+**Example 2 - Calling the hasDetectors Method on the SpectrumInfo Object - Part 1:**
+This example shows how to call the ``hasDetectors`` method.
+The method takes in an integer ``index`` parameter which corresponds to a spectrum.
+The return value is True or False.
+
+.. testcode:: CallHasDetectorsMethod
+
+	# Create a workspace to use (should have a preset instrument)
+	ws = CreateSampleWorkspace()
+
+	# Get the SpectrumInfo object
+	info = ws.spectrumInfo()
+
+	# Call hasDetectors
+	print(info.hasDetectors(0))
+
+Output:
+
+.. testoutput:: CallHasDetectorsMethod
+
+	True
+
+
+**Example 3 - Calling the hasDetectors Method on the SpectrumInfo Object - Part 2:**
```

Comment:
As mentioned in the `DetectorInfo` PR, I think it would be more useful to have a more complete example instead.